planning/behavior_velocity_planner/src/scene_module/speed_bump/util.cpp

```diff
@@ -0,0 +1,281 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <common/types.hpp>
+#include <scene_module/speed_bump/util.hpp>
+#include <utilization/util.hpp>
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/linestring.hpp>
+#include <boost/geometry/geometries/point_xy.hpp>
+
+#include <cmath>
+
+#define EIGEN_MPL2_ONLY
+#include <Eigen/Core>
+
+namespace behavior_velocity_planner
+{
+namespace speed_bump_util
+{
+namespace bg = boost::geometry;
+using Point = bg::model::d2::point_xy<double>;
+using Polygon = bg::model::polygon<Point>;
+using Line = bg::model::linestring<Point>;
+
+bool getBackwardPointFromBasePoint(
+  const Eigen::Vector2d & line_point1, const Eigen::Vector2d & line_point2,
+  const Eigen::Vector2d & base_point, const double backward_length, Eigen::Vector2d & output_point)
+{
+  Eigen::Vector2d line_vec = line_point2 - line_point1;
+  Eigen::Vector2d backward_vec = backward_length * line_vec.normalized();
+  output_point = base_point + backward_vec;
+  return true;
+}
+
+double velocityByBumpHeight(double bump_height)
+{
+  // y = - (1.38/0.20) x + 3.105
+  // y: velocity [m/s]
+  // x: height [m]
+
+  double velocity = 0;
+  if (bump_height <= 0.05) {
+    velocity = 2.76;  // 10 km/h
+  } else if (0.05 < bump_height && bump_height < 0.25) {
+    velocity = -1 * (1.38 / 0.20) * bump_height + 3.105;
+  } else if (bump_height >= 0.25) {
+    velocity = 1.38;  // 5 km/h
+  }
+  return velocity;
+}
+
+bool insertTargetVelocityPoint(
```

Comment:
@beyzanurkaya 
sorry for delay , @purewater0901 -san is uniting many common functios into motion utils so please replace functions
ref  https://github.com/autowarefoundation/autoware.universe/blob/965c0bed48a601b588f885e1e5311c28d88c9537/planning/behavior_velocity_planner/src/scene_module/crosswalk/scene_crosswalk.cpp#L205

Reply:
@taikitanaka3 
Thank you for the comment. I will make the necessary changes.